spec/factories/animals.rb

```diff
@@ -13,17 +12,15 @@
     collection_year { 1 }
     date_time_collected { "2020-04-04 18:52:09" }
     collection_position { "MyString" }
-    pii_tag { 1 }
-    tag_id { 1 }
+    sequence(:tag) { |n| "Y#{n}" }
```

Comment:
So that factory data maps more closely to what we'll see in the real world, how would you feel about left-padding the `n` here? 
```suggestion
    sequence(:tag) { |n| "Y#{sprintf('%03d', 1)}" }
```